tldw_Server_API/app/core/Sharing/unified_share_audit.py

```
@@ -355,6 +354,53 @@ def _build_event(
+        ip_address: str | None = None,
+        user_agent: str | None = None,
+        result: str = "success",
+        created_at: Any = None,
+    ) -> int | None:
+        legacy_id = legacy_share_audit_id
+        if legacy_id is None:
+            legacy_id = legacy_audit_id
+        elif legacy_audit_id is not None and legacy_audit_id != legacy_id:
+            raise ValueError(
+                "legacy_share_audit_id and legacy_audit_id must match when both are provided"
+            )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)